TrackerConditions/inc/StrawElectronics.hh

```diff
@@ -149,7 +143,7 @@ namespace mu2e {
     double digitizationEndFromMarker() const { return _digitizationEnd - _timeFromProtonsToDRMarker; }
     void adcTimes(double time, TrkTypes::ADCTimes& adctimes) const; // given crossing time, fill sampling times of ADC CHECK THIS IS CORRECT IN DRAC FIXME!
     double saturationVoltage() const { return _vsat; }
-    double threshold(StrawId const &sid, StrawEnd::End iend) const { return _vthresh[sid.getStraw()*2 + iend]; }
+    double threshold(StrawId const &sid, StrawEnd::End iend) const { return _vthresh[sid.uniqueStraw()*2 + iend]; }
```

Comment:
Does it make sense, maybe a little cleaner to have a method
sid.UniqueStrawEnd(iend)
?